unity-client/Assets/Scripts/MainScripts/DCL/Controllers/CatalogController/CatalogController.cs

```diff
@@ -12,7 +11,7 @@ public class CatalogController : MonoBehaviour
     private const string BASE_WEARABLES_CONTEXT = "BaseWearables";
     private const int FRAMES_TO_CHECK_FOR_SENDING_PENDING_REQUESTS = 1;
     private const float TIME_TO_CHECK_FOR_UNUSED_WEARABLES = 10f;
-    private const float REQUESTS_TIME_OUT = 5 * 60;
+    private const float REQUESTS_TIME_OUT = 45;
```

Comment:
Can we rename this to include seconds? This way we can be sure in the future about the type

Reply:
Good point! Done in https://github.com/decentraland/explorer/pull/2300/commits/fda9b9cec5a4c0d4e81ce31bda57566f066fa853. Thanks!